src/debug/di/cordb.cpp

```diff
@@ -499,7 +506,7 @@ DbiGetThreadContext(HANDLE hThread,
     DT_CONTEXT *lpContext)
 {
     // if we aren't local debugging this isn't going to work
-#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || defined(__ANDROID__)
+#if !defined(_ARM_) || defined(FEATURE_DBGIPC_TRANSPORT_DI) || !HAVE_ALIGNED_MALLOC
```

Comment:
@parjong On the contrary, the code in the else branch is for local debugging when the debugger code is running in the debugged process. So you could use a define named SUPPORT_LOCAL_DEBUGGING instead.